src/C4Config.cpp

```diff
@@ -547,6 +562,34 @@ void C4ConfigGeneral::DeterminePaths(bool forceWorkingDirectory)
 	}
 	else
 		SCopy("/tmp/", TempPath);
+	StdStrBuf xdg_config_path(getenv("XDG_CONFIG_HOME"));
+	if (!xdg_config_path.getSize()){
+	    xdg_config_path = getenv("HOME");
+	    xdg_config_path += "/.config/legacyclonk";
+	}
+	SCopy(xdg_config_path.getData(), XDG_CONFIG_PATH);
+    SCopy(xdg_config_path.getData(), UserPath);
+    SCopy(xdg_config_path.getData(), LogPath); // I'm not sure if this even gets used, but setting it makes the config look nicer :-)
```

Comment:
Doing so breaks third-party programs assuming the log path to be equal to the executable path. We might not care about it though.